kubectl-rook-ceph.sh

```diff
@@ -263,6 +266,82 @@ function run_purge_osd() {
     rook ceph osd remove --osd-ids=$1 --force-osd-removal=$force_removal"
 }
 
+####################################################################################################
+# 'kubectl rook-ceph debug commands
+####################################################################################################
+
+function update_deployment_spec_command(){
+  deployment_spec=$1
+  main_command='["sleep","infinity"]'
+  deployment_spec=$(jq .template.spec.containers[0].command="$main_command" <<< $deployment_spec)
+  deployment_spec=$(jq .template.spec.containers[0].args=[] <<< $deployment_spec)
+}  
+
+function update_deployment_spec_image(){
+  deployment_spec=$1
+  alternalte_image=$2
+  deployment_spec=$(jq .template.spec.containers[0].image="$alternalte_image" <<< $deployment_spec)
+}
+
+function verify_deployment_exist(){
+  if [[ $deployment_name != *"mon"* ]] && [[ $deployment_name != *"osd"* ]]; then
+    fail_error "only mon or osd deployment name can passed for starting debug mode"
+  fi
+  if output="$($TOP_LEVEL_COMMAND --namespace "$ROOK_OPERATOR_NAMESPACE" get deployment "$deployment_name")"; then
```

Comment:
Should this be `CLUSTER` namespace? I see a lot of instances of the operator namespace, but I don't think we need to do anything in that namespace, right?

```suggestion
  if output="$($TOP_LEVEL_COMMAND --namespace "$ROOK_CLUSTER_NAMESPACE" get deployment "$deployment_name")"; then
```

Reply:
Yaa, it should be cluster_namespace I also think the same, 
But look here https://github.com/rook/kubectl-rook-ceph/blob/256ca021a47939dbe3cae4b954c4d934a61c4956/kubectl-rook-ceph.sh#L335

@subhamkrai your thoughts?

Reply:
change this to cluster namespace since this is osd and mons deploy

Reply:
@parth-gr ^^

Reply:
sure done